doc/source/security/encrypting-tablespaces.rst

```diff
@@ -0,0 +1,63 @@
+.. encrypting-tablespaces:
+
+======================================================
+Encrypting General Tablespaces
+======================================================
+
+.. _innodb_general_tablespace_encryption:
+
+Setting the Default for General Tablespace Encryption
+================================================================================
+
+The encryption of general tablespaces is determined by the :variable: ``default_table_encryption`` variable unless you specify the ENCRYPTION clause in the CREATE TABLESPACE statement.
+
+You can set the :variable: ``default_table_encryption`` variable in an individual connection.
+
+.. code-block:: mysql
+
+   mysql> SET default_table_encryption=ON;
+
+.. variable:: default_table_encryption
+
+    :cli: ``default-table-encryption``
+    :dyn: Yes
+    :scope: Session
+    :vartype: Text
+    :default: OFF
+
+Defines the default encryption setting for general tablespaces. The variable allows you to create or alter tablespaces without specifying the ENCRYPTION clause. The default encryption setting applies only to general tablespaces and is not applied to the mysql system tablespace.
+
+.. seealso::
+
+      MySQL Documentation: default_table_encryption
+      https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html
+
+Setting Tablespace ENCRYPTION without a Default Setting
+---------------------------------------------------------
+
+If you do not set the default encryption setting, you can create general tablespaces with the ``ENCRYPTION`` setting.
+
+.. code-block:: MySQL
+
+  mysql> CREATE TABLESPACE tablespace_name ENCRYPTION='Y';
+
+All tables contained in the tablespace are either encrypted or not encrypted. You cannot encrypted only some of the tables in a general tablespace. This feature extends the  `CREATE TABLESPACE <https://dev.mysql.com/doc/refman/8.0/en/create-tablespace.html>`_ statement to accept the ``ENCRYPTION='Y/N'`` option.
+
+.. note::
+
+   Prior to |Percona Server| 8.0.13, the ``ENCRYPTION`` option was specific to the ``CREATE TABLE`` or ``SHOW CREATE TABLE`` statement.  In |Percona Server| 8.0.13, this option becomes a tablespace attribute and is not allowed with the ``CREATE TABLE`` or ``SHOW CREATE TABLE`` statement except for file-per-table tablespaces.
+
+An attempt to create or to move any tables, including partitioned ones, to a general tablespace with an incompatible encryption setting are diagnosed and and the process is aborted.
+
+If you must move tables between incompatible tablespaces, create tables with the same structure in another tablespace and run ``INSERT INTO SELECT`` from each of the source tables into the destination tables.
+
+.. note::
+
+   |Percona XtraBackup| version 8 supports the backup of encrypted general
+   tablespaces. Any features which are not GA in |Percona Server| are not supported in version 8.
```

Comment:
" Any features which are not GA in |Percona Server| are not supported in version 8." - how is this relevant here?

Reply:
Moved to the general Data at rest encryption doc.